assets/app/view/welcome.rb

```diff
@@ -17,6 +17,7 @@ def render
 
     def render_notification
       message = <<~MESSAGE
+        <p>18CO: Rock & Stock is now in alpha!</p>
```

Comment:
can we consolidate all the other 'alphas' on one line?
drop the 1817 production line.
also need the wiki page for 18CO updating.